### DISCUSSION TITLE
Please remove requirement that end time is before start time - Fox have fixed/removed this

### DISCUSSION THREAD

**Discussion by iancg** [nathanmarlor/foxess_modbus#413] [Ideas]
Hi,

Please see:

https://www.facebook.com/groups/foxessownersgroup/posts/1923430298039143/?comment_id=1923434278038745&reply_comment_id=1924573837924789

I've tested this and with software Master 1.60/Manager 1.63, using start=23:30 and end=05:30 worked as expected.

Can the check that prevents start > end be removed please.

Setting on web UI:

![image](https://github.com/nathanmarlor/foxess_modbus/assets/1828481/3445bfad-8723-470b-b421-3da0d165af37)

Battery charged overnight (it was full just before 3am, but started at 11:30 and continued over midnight and didn't start discharging until 5:30am).

![image](https://github.com/nathanmarlor/foxess_modbus/assets/1828481/c43bf310-a8cf-47af-a31d-7f29797a744b)

**Comment by canton7** [comment]
Opened #414

**Comment by FozzieUK** [comment]
Good spot !, I wonder when that change crept in.

Tried it last night, and worked perfectly.

![IMG_7975](https://github.com/nathanmarlor/foxess_modbus/assets/113460294/eb2be64a-596e-4772-b234-0a7df49daf3b)
